src/backend/src/prisma/schema.prisma

```diff
@@ -314,3 +317,71 @@ model Task {
   wbsElement      WBS_Element   @relation(fields: [wbsElementId], references: [wbsElementId])
   wbsElementId    Int
 }
+
+enum Club_Accounts {
+  CASH
+  BUDGET
+}
+
+enum Reimbursement_Status_Type {
+  PENDING_FINANCE
+  SABO_SUBMITTED
+  ADVISOR_APPROVED
+  REIMBURSED
+}
+
+model Reimbursement_Status {
+  reimbursementStatusId  Int                       @id @default(autoincrement())
+  type                   Reimbursement_Status_Type
+  userId                 Int
+  user                   User                      @relation(fields: [userId], references: [userId])
+  dateCreated            DateTime                  @default(now())
+  reimbursementRequestId String
+  reimbursementRequest   Reimbursement_Request     @relation(fields: [reimbursementRequestId], references: [reimbursementRequestId])
+}
+
+model Reimbursement_Request {
+  reimbursementRequestId String                  @id @default(uuid())
+  saboId                 Int?
+  dateCreated            DateTime                @default(now())
+  dateDeleted            DateTime?
+  dateOfExpense          DateTime
+  reimbursementsStatuses Reimbursement_Status[]
+  recepientId            Int
+  recepient              User                    @relation(name: "reimbursementRequestRecepient", fields: [recepientId], references: [userId])
+  vendorId               String
+  vendor                 Vendor                  @relation(fields: [vendorId], references: [vendorId])
+  account                Club_Accounts
+  totalCost              Int
+  receiptPicture         String[]
```

Comment:
also the more I'm looking into this, this is not the best way to store images. We'll probably wanna upload it to a file storage system, I think we might be able to use google drive, or we could definitely use this thing aws has called aws s3